ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -24,22 +26,22 @@
  */
 public class BlobPropertiesSerDe {
 
-  static final short Version1 = 1;
-  static final short Version2 = 2;
-  private static final short currentVersion = Version1;
-  private static final int Version_Field_Size_In_Bytes = 2;
-  private static final int TTL_Field_Size_In_Bytes = 8;
-  private static final int Private_Field_Size_In_Bytes = 1;
-  private static final int CreationTime_Field_Size_In_Bytes = 8;
-  private static final int Variable_Field_Size_In_Bytes = 4;
-  private static final int BlobSize_Field_Size_In_Bytes = 8;
+  static final short VERSION_1 = 1;
+  static final short VERSION_2 = 2;
+  private static final short CURRENT_VERSION = VERSION_2;
```

Comment:
Yes. But thats for readability and documentation purposes to quickly figure out the current version in which serialization happens.